packages/hd/src/keystore.ts

```diff
@@ -151,19 +144,17 @@ export default class Keystore {
       )
     );
 
-    const cipher: Cipher = createCipheriv(CIPHER, derivedKey.slice(0, 16), iv);
-    if (!cipher) {
-      throw new UnsupportedCipher();
-    }
-
-    // size of 0x prefix
-    const hexPrefixSize = 2;
-    const ciphertext: Buffer = Buffer.concat([
-      cipher.update(
-        Buffer.from(extendedPrivateKey.serialize().slice(hexPrefixSize), "hex")
-      ),
-      cipher.final(),
-    ]);
+    // DO NOT remove the Uint8Array.from call below.
+    // Without calling Uint8Array.from to make a copy of iv,
+    // iv will be set to 0000...00000 after calling cipher.encrypt(plaintext)
+    // and decrypting the ciphertext will fail
+    /* eslint-disable @typescript-eslint/no-magic-numbers */
```

Comment:
i found that the number 2 is common in Lumos since the `str.slice(2)` is used often, so i'd suggest adding the 2 to ignore list to avoid the disable annotation

Reply:
Added (back).
It was there before. 

In `.eslintrc.next.js`
```ts
        "@typescript-eslint/no-magic-numbers": [
          "error",
          {
            ignoreEnums: true,
            ignoreArrayIndexes: true,
            ignoreClassFieldInitialValues: true,
            ignoreDefaultValues: true,
            ignore: [
              -1, // index -1 is not found
              0, // first element of an array
              1, // common for i + 1 in a loop
              2, // many .slice(2) since the '0x' prefix should be removed while calling 3rd-party library
              16, // toString(16)
              1000, // second to millisecond
            ],
          },
        ],
```